### ISSUE TITLE
Hide revealed Attributes in Proof Presentation

### ISSUE THREAD

**Issue by PenguinTaro** [hyperledger/aries-framework-javascript#1423]
Hi there, In our use case we want to set the revealed flag to false so that verifier doesn't receive the details of my attributes. Can you guide me regarding this, like how this can be achieved?
Thank you.

**Comment by TimoGlastra** [comment]
I don't think that flow is currently worked out. I think we should allow a variant of the selected credentials where `credentialId` is undefined and `revealed` is false. 

It would require changes to the anoncreds credential format and anoncreds services though. Are you willing to make a PR for this?
